src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl_norels.yaml

```diff
@@ -0,0 +1,43 @@
+metadata:
+  title: Learn CoDEx-Large Dataset with NodePiece + RotatE (no relational tokenization) as described by Galkin et al., 2022
+pipeline:
+  dataset: CoDExLarge
+  evaluator_kwargs:
+    filtered: true
+  loss: BCELoss
+  model: NodePiece
+  model_kwargs:
+    entity_initializer: xavier_uniform_
+    relation_initializer: init_phases
+    relation_constrainer: complex_normalize
+    embedding_dim: 200   # for both anchors and relations
+    interaction: RotatEInteraction
+    aggregation: mlp
+    tokenizers: AnchorTokenizer
+    num_tokens: 20  # 20 anchors
+    tokenizers_kwargs:
+      [{
+         selection: MixtureAnchorSelection
+         searcher: CSGraphAnchorSearcher
+         selection_kwargs:
+           selections: ['Degree', 'PageRank', 'Random']
+           ratios: [0.4, 0.4, 0.2]
+           num_anchors: 7000
+      },
+       {}  # no params for RelationTokenizer
```

Comment:
superfluous? This is a configuration without relation tokenizer

Reply:
related to the `make_many` thing as the above comment ^